persistence/persistence.go

```diff
@@ -35,6 +35,7 @@ var (
 	ErrConstraintFailed       = errors.New("constraint failed")
 	ErrUniqueConstraintFailed = errors.New("unique constraint failed")
 	ErrUnsupportedType        = errors.New("unsupported type")
+	ErrDatabase               = errors.New("database error")
```

Comment:
What is this used for?

Reply:
It is the error message for general database errors.